src/components/ListOfClubs/ListOfClubs.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import styled from "styled-components";
+
+
+
+const ColumnTitle = styled.p`
+    font-weight: bold;
+`;
+const Row = styled.div`
+    /* display: flex; */
+    /* flex-direction: row; */
+    display: grid;
+    grid-template-columns: 25% 20% 15% 20% 10% 10%;
+`;
+const Column = styled.p`
+    /* width: 20%; */
+    margin: 2% 0;
+    padding: 0 1% 1% 0;
+`;
+const ActionColumn = styled.p`
+    margin: 2% 0;
+    padding: 0 1% 1% 0;
+    text-decoration: underline;
+    cursor: pointer;
+`;
+const DownwardArrow = styled.i`
+    cursor: pointer;
+    padding-left: 1%;
+`;
+
+const MoreInfo = styled.div`
+    display: none;
+    padding: 0 0 0 1%;
+    margin: 1% 0 2% 0;
+    div{
+        display: grid;
+        grid-template-columns: 8% 88%;
+        font-size: 90%;
+        p{
+            margin: 0;
+            padding: 11% 0 ;
+        }
+        span{
+            padding: 1% 0 1% 2%;
+        }
+    }
+`;
+export const ListOfClubs = ({clubs,columnTitles}) => {
+    // const [moreInfo, setMoreInfo] = useState(false);
+    const expandInfo = (index)=>{
+        var isDisplayed = document.getElementById(`more-info-${index}`).style.display;
+        isDisplayed ? isDisplayed==="none" ? document.getElementById(`more-info-${index}`).style.display= "inherit" 
```

Comment:
- Although in React we generally don't manipulate the DOM directly, I really like your approach here!
- For code clarity, it might be better to write it as follows:
```javascript
/* Your logic
if ( isDisplayed ) {
   if ( isDisplayed === "none" ) 
       isDisplayed = "inherit";
   else
       isDisplayed = "none";
} else {
   isDisplayed = "none";
}
*/

/* from above, it seems that it can be reduced to the following */
isDisplayed = isDisplayed === 'none' ? 'inherit' : 'none';
```